components/TInput/TInputProfile.vue

```diff
@@ -68,39 +70,47 @@
       </div>
       <template v-if="inviteUsername">
         <div
-          class="flex px-4 py-2 hover:bg-blue-100 items-center cursor-pointer space-x-1 text-sm text-gray-700"
-          @click="
-            create({
-              username: inviteUsername,
-              instagram: `https://instagram.com/` + inviteUsername,
-            })
-          "
+          class="flex justify-center px-4 py-2 hover:bg-blue-100 items-center cursor-pointer space-x-1 text-sm text-gray-700"
         >
-          <div>
-            {{ $t('TInputProfile.import') }}
-          </div>
-          <TIcon name="instagram" size="4" />
-          <div>
-            {{ inviteUsername }}
-          </div>
+          <h3 @click="showPopup = !showPopup" class="text-center">
+            + Add New Organizer
+          </h3>
         </div>
-        <div
-          class="flex px-4 py-2 hover:bg-blue-100 items-center cursor-pointer space-x-1 text-sm text-gray-700"
-          @click="
-            create({
-              username: inviteUsername,
-              facebook: `https://facebook.com/` + inviteUsername,
-            })
-          "
-        >
-          <div>
-            {{ $t('TInputProfile.import') }}
+        <TPopup v-if="showPopup">
+          <div class="flex justify-between border-b pb-2 mb-4">
+            <div class="grow font-bold text-center ">
+              Add New Organizer
+            </div>
+            <button class=" cursor-pointer" @click="showPopup = false">
+              <TIcon name="close" class="cursor-pointer w-4 h-4" />
+            </button>
           </div>
-          <TIcon name="facebook" size="4" />
-          <div>
-            {{ inviteUsername }}
+          <div class="my-4 flex flex-col justify-center">
+            <div>Social Media or Website</div>
+            <input
```

Comment:
It should be repeatable field. 

As soon as user fills it, a new field should be shown below.

See like Artists field in the Event Editor.

It is similar to XArray.

Reply:
solved